### ISSUE TITLE
LogPosition command

### ISSUE THREAD

**Issue by ripSquid** [kami-blue/client#713]
**Is your feature request related to a problem? Please describe.**
Not being able to find back to my base

**Describe the solution you'd like**
A command which logs the coords attached with a name to a file somewhere.
Usage would be basically ;logpos name . Where name is the name attached to the coords.

Adding to this. Maybe have a command to display names of saved coords and then by specifying name you can get coords of that. So listpos will display a list of saved positions and listpos name would display that coords

**Describe alternatives you've considered**
Basically saving them manually. But this would help much as it would be fast.
